### ISSUE TITLE
您好，请问beam_search等超参应该是没有用的吧

### ISSUE THREAD

**Issue by leoozy** [bytedance/lightseq#121]
这些参数应该是模型输出了vocabulary分布之后再做处理的，还是说模型包含了对分布的处理？

**Comment by leoozy** [comment]
或者说解码部分我可以不使用ls么，我只需要bart之类模型输出的vocabulary分布

**Comment by neopro12** [comment]
Logits in every step is the output of decoder/model and the input of search module(like beam search). Currently, LightSeq can't output logits to user. But logit is available here: https://github.com/bytedance/lightseq/blob/master/lightseq/inference/model/decoder.cc.cu#L359. 
You may need some secondary development if you want it now.

**Comment by neopro12** [comment]
We will separate the encoder/decoder/search module and wrap them as independent python functions in the near future.

**Comment by leoozy** [comment]
@neopro12 Looking forward to your works. The custum decoding methods suffer a lot of practical problems when deploying them. Self-defined decoding methods needs the logits

**Comment by Taka152** [comment]
@leoozy `lightseq.training` can accelerate the encoder layer and decoder layer without searching. You can try it.
